www/include/configuration/configObject/traps/formTraps.php

```diff
@@ -321,7 +352,17 @@ function myReplace()
 
 $form->addElement('textarea', 'traps_customcode', _("Custom code"), $attrsTextarea);
 
-$form->addElement('select', 'traps_advanced_treatment_default', _("Advanced matching behavior"), array(0=>_("If no match, submit default status"), 1=>_("If no match, disable submit"), 2=>_("If match, disable submit")), array('id' => 'traps_advanced_treatment'));
+$form->addElement(
+    'select',
+    'traps_advanced_treatment_default',
+    _("Advanced matching behavior"),
+    array(
+        0=>_("If no match, submit default status"),
+        1=>_("If no match, disable submit"),
+        2=>_("If match, disable submit")
```

Comment:
Add whitespace before & after the key, value
```suggestion
         2 => _("If match, disable submit")
```